pom.xml

```diff
@@ -208,6 +211,25 @@
                     </configLocation>
                 </configuration>
             </plugin>
+            
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>findbugs-maven-plugin</artifactId>
+                <version>${findbugs-maven-plugin.version}</version>
+                <configuration>
+                    <xmlOutput>true</xmlOutput>
+                    <failOnError>${findbugs.failOnError}</failOnError>
```

Comment:
-Dfindbugs.failOnError can be redefined from the external run with such approach even if you don't specify this line explicitly

If you disagree with this string, I'll remove it